os/board/rtl8720e/src/component/soc/amebalite/fwlib/ram_km4/ameba_ota_ram.c

```diff
@@ -480,5 +480,41 @@ int OTA_Image2SignatureCheck(uint32_t input_addr)
 	return TRUE;
 }
 
+/* Secure Key Derive Function structure */
+typedef struct {
+	unsigned char *secret;
+	size_t slen;
+	char *label;
+	unsigned char *context;
+	size_t clen;
+	unsigned char *dstbuf;
+	size_t dlen;
+} secure_kdf_struc;
+
+int Ameba_KeyDeriveFunc(const unsigned char *secret, size_t slen,
+			const char *label,
+			const unsigned char *context, size_t clen,
+			unsigned char *dstbuf, size_t dlen)
```

Comment:
Because I am not in charge of security, it could be an unnecessary question.
When security guy checks the API, all of arguments are easy understandable?
If not, could you add some description for argus?

Reply:
Updated a new API.